components/automate-cli/pkg/diagnostics/integration/iam_projects.go

```diff
@@ -130,16 +130,19 @@ func GetProject(tstCtx diagnostics.TestContext, id string) (*ProjectInfo, error)
 	return &projectInfo, nil
 }
 
-// DeleteProject deletes the project with the given id
-func DeleteProject(tstCtx diagnostics.TestContext, id string) error {
-	// any associated rules are cascade-deleted with the project
```

Comment:
Turns out this wasn't the case. It return `err == nil`, but `resp.StatusCode == 400` because the project had a rule.